addons/web/static/src/js/views/list/list_controller.js

```diff
@@ -99,6 +99,33 @@ var ListController = BasicController.extend({
             return record.res_id;
         });
     },
+
+    /**
+     * @typedef {Object} SelectedIds
+     * @property {integer[]} resIds
+     * @property {integer} total - The total number of selected records
+     * @property {boolean} displayNotif - True if not all the available records have been returned
+     */
+
+    /**
+     * Returns the list of currently selected res_ids (with the check boxes on
+     * the left) or the whole domain res_ids if it is selected
```

Comment:
Maybe indicate in the docstring that this should actually be the implementation of `getSelectedIds`, but we kept it for now for backward compatibility reasons ?